main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -1685,8 +1685,8 @@ object Parser2 {
         case TokenKind.KeywordDebug
              | TokenKind.KeywordDebugBang
              | TokenKind.KeywordDebugBangBang => debugExpr()
-        case TokenKind.KeywordXmatch => extensibleMatchExpr()
-        case TokenKind.KeywordXvar => extensibleTagExpr()
+        case TokenKind.KeywordEMatch => extMatchExpr()
```

Comment:
Keep sorted

Reply:
None of these are sorted. Maybe in another pr?